Add project: P300 EEG ERP Analysis by Fabiana Ojeda

faojeda opened the pull request:
This pull request adds my Brainhack School 2025 project.

- Added my project markdown file in `content/en/project/`
- Included three figures: `erp_comparison.png`, `difference_wave.png`, and `thumbnail.png` in `static/project/img/`
- Linked my GitHub repo and slides
- Render-ready and complete

Thanks for reviewing!

lunebellec commented on static/project/img/difference_wave.png:
please move this image in the same folder as the project/index.md file.

lunebellec commented on static/project/img/erp_comparison.png:
please move this image in the same folder as the project/index.md file.

lunebellec commented on static/project/img/thumbnail.png:
please move this image in the same folder as the project/index.md file.

lunebellec commented on content/en/project/p300-eeg-fabiana/index.md:
```diff
@@ -0,0 +1,75 @@
+
+summary: "This project explores how deviant auditory tones in a cross-modal oddball paradigm elicit a stronger P300 component using EEG data from the MNE sample dataset. The analysis focuses on ERP comparison and difference waves, setting the stage for future investigations on emotional modulation of P300."
+
+image: "project/img/thumbnail.png"
```
```suggestion
image: "thumbnail.png"
```

faojeda commented:
Thanks @lunebellec for the review! I’ve moved all images to the project folder and updated the markdown as requested. Let me know if anything else needs tweaking.

lunebellec commented:
LGTM!! Congrats on publishing your project report 🎉
